core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/kubernetes/KubernetesClient.scala

```diff
@@ -176,6 +183,11 @@ class KubernetesClient(
         waitForPod(namespace, createdPod, start.start, config.timeouts.run)
           .map { readyPod =>
             transid.finished(this, start, logLevel = InfoLevel)
+            val t1 = System.currentTimeMillis()
+            MetricEmitter.emitHistogramMetric(
+              LogMarkerToken("kubeapi", "create", "duration", Some("create"), Map("cmd" -> "create"))(
+                MeasurementUnit.time.milliseconds),
```

Comment:
Currently, just add kubeapi for create, maybe we can add kubeapi for delete in future.

Reply:
The related transaction already exists. And it is better to use transid than to calculate time directly.

```
    val start = transid.started(
      this,
      LoggingMarkers.INVOKER_KUBEAPI_CMD("create"),
      s"launching pod $name (image:$image, mem: ${memory.toMB}) (timeout: ${config.timeouts.run.toSeconds}s)",
      logLevel = akka.event.Logging.InfoLevel)
```

Reply:
Updated accordingly.

Reply:
`transid.finished` method already sends Histogram metric here. What's the difference?

https://github.com/apache/openwhisk/blob/99fb2c59e264d820d6b95a6319b7882f19aead4d/common/scala/src/main/scala/org/apache/openwhisk/common/TransactionId.scala#L125

Reply:
@upgle ,good found, has no any difference, i already removed the duplicated one, just keep `transid.finish` to send the metric.